.github/workflows/ci.yml

```diff
@@ -46,6 +34,24 @@ jobs:
           - name: CPython 3.9
             tox: py39
             action: 3.9
+          - name: CPython 3.10
+            tox: py310
+            action: '3.10'
+        include:
+          - os:
+              name: Windows
+              runs-on: windows-latest
+            python:
+              name: CPython 3.6
+              tox: py36
+              action: 3.6
+          - os:
+                name: Windows
+                runs-on: windows-latest
+            python:
+                name: CPython 3.9
+                tox: py39
+                action: '3.9'
```

Comment:
There are things changing in typing at boundaries in between the oldest and latest supported versions.  It's also simpler to just test everything everywhere I think, unless we have insufficient parallelism right now for all the jobs?

Reply:
Ok. I don't like using too much power in general, hence the attempt to be reasonably minimalistic.

Reply:
That's good.  Don't forget the relatively insane cost of an undetected bug.  `:]`